Mathlib/Topology/Order/IntermediateValue.lean

```diff
@@ -486,10 +484,32 @@ theorem intermediate_value_Icc' {a b : α} (hab : a ≤ b) {f : α → δ}
   isPreconnected_Icc.intermediate_value (right_mem_Icc.2 hab) (left_mem_Icc.2 hab) hf
 
 /-- **Intermediate Value Theorem** for continuous functions on closed intervals, unordered case. -/
-theorem intermediate_value_uIcc {a b : α} {f : α → δ} (hf : ContinuousOn f (uIcc a b)) :
-    uIcc (f a) (f b) ⊆ f '' uIcc a b := by
+theorem intermediate_value_uIcc {a b : α} {f : α → δ} (hf : ContinuousOn f [[a, b]]) :
+    [[f a, f b]] ⊆ f '' uIcc a b := by
```

Comment:
Is there a reason you're not writing `f '' ([[a, b]])` on the right hand side? The fact that you need brackets?

Reply:
The only reason is that I was writing it at about 1am and missed this `uIcc`.